lib/pharos/resources/openebs/30-default-storage-pool.yml.erb

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: openebs.io/v1alpha1
+kind: StoragePool
+metadata:
+  name: default
+  type: hostdir
+spec:
+  path: <%= default_storage_pool_path %>
+  nodes: [ "all" ]
+  #maxVolSize: 10G #Max Volume Size
```

Comment:
Why are these commented out?

Reply:
I not 100% sure how they affect, so had them still left commented out there. Probably there should be some config option for these.

Reply:
removed them now. we could add more options in later PR's, IMO no need to put everything into this initial PR.